L3-L4-OOP/L4-Zad.6-BillCalculator/src/main/java/pl/babiarz/BillCalculator.java

```diff
@@ -0,0 +1,13 @@
+package pl.babiarz;
+
+public class BillCalculator {
+    public static double calculate(double billValue, float serviceFee) {
+        return billValue + serviceFee;
+    }
+    public static double calculateWithDiscount(double billValue, float serviceFee, double discount){
+        return (billValue - discount) + serviceFee;
+    }
+    public static double calculateWithPackagingFee(double billValue, float serviceFee, short PackagingFee) {
```

Comment:
co to za dziwna nazwa zmiennej 'PackingFee' - czemu z wielkiej litery? Tak piszemy zmienne/argumenty metody?